torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1229,19 +1229,29 @@ struct PythonPrintImpl {
       // attributes and parameters.
       if (is_module) {
         std::vector<std::string> params;
+        std::vector<std::string> buffers;
```

Comment:
Nice! Before committing this, we'll have to test for forward compatibility. Basically, can you `save()` with buffers, then `load()` using a version of pytorch that does not understand buffer serialization.

Based on skimming the import code, I think there will be an FC issue. Let's chat directly about the steps to take here.

Reply:
Agreed, let me figure out the best way to do so.